include/mbedtls/ssl.h

```diff
@@ -3751,6 +3751,8 @@ void mbedtls_ssl_conf_groups(mbedtls_ssl_config *conf,
  *                 used for certificate signature are controlled by the
  *                 verification profile, see \c mbedtls_ssl_conf_cert_profile().
  *
+ * \deprecated     Superseded by mbedtls_ssl_conf_sig_algs().
```

Comment:
That's what I suspected, but the documentation for `conf_sig_algs()` seems to suggest it's only for TLS 1.3. If it's actually common to 1.3 and 1.2, then its documentation should be edited to reflect that. Not just the `\brief` part but also `\param sig_algs` which only mentions 1.3 macros/values. IIRC the 1.3 values are purposefully compatible with 1.2 values but that probably needs explaining.

Reply:
Yes. I will add those later.